benches/benches/transaction_throughput.rs

```diff
@@ -281,17 +288,90 @@ fn predicate_transfers_eck1(c: &mut Criterion) {
             .add_output(Output::coin(rng.gen(), 50, AssetId::default()))
             .add_output(Output::change(rng.gen(), 0, AssetId::default()))
             .finalize();
-        tx.estimate_predicates(&CheckPredicateParams::default(), MemoryInstance::new())
+        tx.estimate_predicates(&checked_parameters(), MemoryInstance::new())
             .expect("Predicate check failed");
         tx
     };
     bench_txs("predicate transfers eck1", c, generator);
 }
 
+fn predicate_transfers_ed19(c: &mut Criterion) {
```

Comment:
This function looks very similar to `predicate_transfers_eck1`. I assume the intention is for them to be exactly the same but with different signing schemes. It would be nice to have the code reflect this by breaking out the shared logic to a separate function.

If you agree, I'd be happy to create a ticket and do that as a follow-up.

Reply:
Exactly. The idea here is to benchmarks a bunch of simple transactions to get an idea of max throughput for our system. In this case we are including a predicate that includes
```
    op::ed19(0x20, 0x21, 0x22, 0x24),
```
The entry point for these benchmarks is the `criterion_group!` macro call below. I think the differences are moderately understandable looking there:
```
criterion_group!(
    benches,
    signed_transfers,
    predicate_transfers,
    predicate_transfers_eck1
    predicate_transfers_ed19
);
```
But I'll never argue with adding more clarity to code if things are unclear.

Reply:
Right, thanks for clarifying. As long as it's just two signature schemes, a refactor of these functions may be a bit premature.